sc/shared/src/test/scala/sigmastate/utxo/BasicOpsSpecification.scala

```diff
@@ -144,6 +147,77 @@ class BasicOpsSpecification extends CompilerTestingCommons
     flexVerifier.verify(verifyEnv, tree, ctxExt, pr.proof, fakeMessage).get._1 shouldBe true
   }
 
+  property("Context.getVar") {
+    def varTest() = {
+      test("GetVar1", env, ext,
+        "{ CONTEXT.getVar[Int](intVar2.toByte).get == 2 }",
+        null
+      )
+    }
+
+    if(VersionContext.current.isV6SoftForkActivated) {
+      varTest()
+    } else {
+      an[Exception] should be thrownBy(varTest())
+    }
+  }
```

Comment:
Missing test case for different actual and expected types.

Reply:
Reworked into more nuanced check (for sigma.validation.ValidationException now)